src/Makefile

```diff
@@ -84,6 +84,7 @@ else
     endif
 
     LIBS += $(shell $(PKG_CONFIG) $(PKG_CONFIG_FLAGS) --libs ncursesw)
+    CPPFLAGS += -D'KAK_BIN_PATH="$(bindir)/kak"'
```

Comment:
Are the simple quotes necessary here?

Reply:
The single quotes can be replaced with escaped double quotes: -DKAK_BIN_PATH=\"$(bindir)/kak\". It's a matter of preference.

I used single quotes because that's how it was already done. It can be changed to use escaped double quotes for both occurrences if wanted.

Reply:
```
-DKAK_BIN_DIR="$(bindir)/kak"
```

The above works on my system, does it not with BSD/make as well? I can see now that the same syntax (with both styles of quotes) is being used for OpenBSD targets, it's not clear to me why you would need to escape the expansion twice.

Reply:
Escaping is necessary because the quotes are part of the define value, and are not intended to be parsed by the shell.

Without escaping the path, the preprocessor will output invalid C++: `return /usr/pkg/bin/kak`.

Reply:
> ```
> -DKAK_BIN_DIR="$(bindir)/kak"
> ```
> 
> The above works on my system, does it not with BSD/make as well? I can see now that the same syntax (with both styles of quotes) is being used for OpenBSD targets, it's not clear to me why you would need to escape the expansion twice.

The only reason it "works on my system" is that you're not running NetBSD and so the code isn't actually getting compiled.

Reply:
Running NetBSD is irrelevant, it was a matter of not letting the shell expand the string as @niacat explained.

Reply:
It's actually very relevant; anything inside an #if block isn't going to be seen by the C compiler if the conditional evaluates to 0. And if the code doesn't get seen by the C compiler, it can't see the syntax error you've just introduced by stripping out the quotes.

Reply:
I'm running Linux, I could test this PR without having to use NetBSD - it's irrelevant.

Thanks @niacat for the explanation, sorry for the noise!